tests-arquillian/src/test/java/org/jboss/weld/tests/extensions/annotatedType/withAnnotations/WithAnnotationsTest.java

```diff
@@ -57,4 +59,12 @@ public void test() {
         assertNotNull(extension.getGroupType());
         assertEquals(Group.class, extension.getGroupType().getJavaClass());
     }
+    
+    // We need to update WildFly ClassFileInfo first
```

Comment:
Maybe we can have this test as a separate PR (and not just for embedded container) and put it on hold until we fix this on WFLY side. CI is using nightly builds, so once we fix it there, we can easily re-run the PR and see the results.

Reply:
Well, that was just really an idea so that we don't forget to enable this test on WFLY later on.